COMM:
    type: 1                 # CAN_PEAK_USB
    path: /dev/pcan32
    baudrate: 0x0           # CANITFBAUD_250K

FTS:
    base_identifier: 0x20
    future_baudrate: 1      # ATI_CAN_BAUD_1M
    future_base_id: 0x20
